tools/lint-pr-commit-message.sh

```diff
@@ -25,21 +25,15 @@ if [ -z "${PR_ID}" ]; then
   echo " e.g. $0 <PR_NUMBER>"
   exit 1
 fi
-# Retrieve the first commit of the pull request via GitHub API
-# TODO: If we teach core-validate-commit to ignore "fixup!" and "squash!"
```

Comment:
Isn't this comment still relevant? The only thing that's different in it is the url in the command line example?

Reply:
Ah, you're right, I didn't read all of it and assumed it was just talking about switching to `npx -q core-validate-commit --no-validate-metadata` which is already there. It's still not quite right because it's clear we can't continue to use the API. `core-validate-commit` is going to have to be fed the list of commits.

Reply:
No objections to removing this TODO comment.